selftests/unit/test_mux.py

```diff
@@ -312,7 +312,7 @@ def test_basic(self):
         self.assertNotEqual(self.params1, self.params2)
         repr(self.params1)
         str(self.params1)
-        str(varianter.AvocadoParams([], 'Unittest', [], {}))
+        str(parameters.AvocadoParams([], 'Unittest', [], {}))
```

Comment:
ok, no actual change of behavior. The module split makes sense even if we hadn't further goals.